string ee_id
geometry_msgs/PoseStamped[] grasp_poses
float32[] grasp_ranks
